car_service/request_service.rb

```diff
@@ -0,0 +1,118 @@
+require_relative 'request_repo'
+
+#computed in seconds
+module TimeUtils
+  HOUR = 3600
+  DAY = 24 * 3600
+  MINUTE = 60
+end
+
+#schedule of the car wash
+module Schedule
+  OPENING_TIME = 8
+  CLOSING_TIME = 18
+  OPENING_TIME_WEEKEND = 9
+  CLOSING_TIME_WEEKEND = 15
+end
+
+class RequestService
+  include Schedule
+  include TimeUtils
+
+  def initialize
+    #one for each worker, 2 cars can be washed at the same time
+    @schedule1 = Repo.new
+    @schedule2 = Repo.new
+  end
+
+  # @param [Time] time
+  def add_request
+    new_request = Request.new Time.now
+    set_pickup_date new_request
```

Comment:
You're not consistent with parentheses, you don't have them here, but you use them at line `#37`. You can check the preferred style guide here -> https://github.com/rubocop-hq/ruby-style-guide#method-invocation-parens